dtp/src/utils.rs

```diff
@@ -353,11 +276,12 @@ pub(crate) async fn gen_wallets_transfer_tssc(
     funder_wallet: Wallet<SigningKey>,
     funding_amount: u64,
     fund_contract_addr: Address,
+    chain_id: u64,
 ) -> eyre::Result<Vec<Wallet<SigningKey>>> {
     // generate some new accounts and send funds to each of them
-    let mut wallet_addresses = Vec::<Address>::new();
-    let mut wallet_priv_keys = Vec::<String>::new();
-    let mut signers: Vec<Wallet<SigningKey>> = Vec::new();
+    let mut wallet_addresses = Vec::<Address>::with_capacity(num_accounts as usize);
```

Comment:
Generally speaking I like to avoid mutation which I believe could be done here with a `map` function.

Reply:
[commit](https://github.com/abhi3700/domain-transaction-producer/pull/5/commits/f0dc790970d039fca4c54e02af52285c1574ab3a)